Assets/HoloToolkit/UX/Scripts/Pointers/ControllerPointerBase.cs

```diff
@@ -0,0 +1,166 @@
+﻿using HoloToolkit.Unity;
+using HoloToolkit.Unity.InputModule;
+using System;
+using UnityEngine;
+using UnityEngine.EventSystems;
+using UnityEngine.XR.WSA.Input;
+
+namespace HoloToolkit.Unity.UX
```

Comment:
Should this actually be in UX or InputModule?

Reply:
Not sure. There will be multiple pointer types with different looks, shapes, capabilities, etc. So we're kind of veering into design / UX territory rather than core functionality.